modules/seqexec/server/src/main/scala/seqexec/server/ghost/GHOSTController.scala

```diff
@@ -155,20 +194,20 @@ object GHOSTController {
     implicit val eq: Eq[GHOSTConfig] = Eq.by(
```

Comment:
Is this Eq law-tested?

Reply:
Oh, are you talking about `SeqexecServerArbitraries`? Because if so, there is an arbitrary for `GHOSTConfig`.

Reply:
Never mind :-). I do appear to have a `GHOSTSpec` that does that. I must have added it in way back.